docs/src/tutorials/discrete_stochastic_example.md

```diff
@@ -1,135 +1,297 @@
-# Discrete Stochastic (Gillespie) Equations
+# Continuous-Time Jump Processes 
```

Comment:
I feel like we need to put Gillespie in there just for the search results. Continuous-Time Jump Processes (Gillespie) or something.

Reply:
Yeah I'll update that. I noticed a few other tweaks on the JumpProblem page to make too (I only want to expose the new MassActionJump interface in the docs).